person.rb

```diff
@@ -1,21 +1,23 @@
-require_relative 'nameable'
-
-class Person < Nameable
-  attr_reader :id
+class Person
+  attr_reader :id, :rentals
```

Comment:
* Kindly Create the has-many side of Person and Rental (a person has many rentals) by making rentals `attr_accessor` as stated in the [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20has%2Dmany%20side%20of%20Person%20and%20Rental%20(a%20person%20has%20many%20rentals).)